src/services/dispatchers/actions/types.ts

```diff
@@ -65,8 +65,14 @@ export enum ActionType {
   TASK = 'task',
 }
 
-export interface MailAction {
-  id?: ObjectId;
+export interface ActionBase {
+  id: ObjectId;
+  /** The unique name of the dispatcher */
```

Comment:
`The unique name of the dispatcher` -> `The unique name of the Action`

Same for description? 🤔